### ISSUE TITLE
Improve Non-Rule readability

### ISSUE THREAD

**Issue by dksmiffs** [isocpp/CppCoreGuidelines#1356]
Format suggestion:  "NR.#: Don't: [Active verb] [rest of non-rule here]"

Suggest rewording the following non-rules' titles to make them more readable:

- NR.1: Don't: Place all declarations at the top of a function
- NR.3: Don't: Avoid exceptions
- NR.5: Don't: Avoid substantive work in a constructor; instead use two-phase initialization

**Comment by cubbimew** [comment]
I agree that double negation is confusing.

**Comment by hsutter** [comment]
Editors call: We agree these could be reworded better to avoid double negation.
